app/models/user.rb

```diff
@@ -2,6 +2,8 @@ class User < ApplicationRecord
 
   has_secure_password
 
+  has_many :counselors
```

Comment:
It should; within the users table we'd have more than one user who occupies the role of "counselor", however each counselor belongs to a user.